source/manual/running-rake-tasks.html.md

```diff
@@ -21,24 +21,9 @@ kubectl -n apps exec deploy/publishing-api -- rake 'represent_downstream:publish
 
 The output of the command will be streamed to your terminal.
 
-## Run a rake task on EC2
+## Run rake tasks from the command line
```

Comment:
```suggestion
## Run rake tasks on EC2 from the command line
```

Just to clarify the difference with "Run a rake task on EKS" (which also uses the command line)

Reply:
Ah, I was too slow. No worries, fine as it is :)